CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# How to Contribute to BotKube
+
+We'd love your help!
+
+BotKube is [MIT Licensed](LICENSE) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
```

Comment:
add link to BotKube docs here.

Reply:
I have added a section about contributing to docs